### PR TITLE
Add tester trainer role

### PR DESCRIPTION
**Pull request by leefaisonr**
Ref #618

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -105,6 +105,8 @@ def eligible_sysadmin?
     return true if superuser || sysadmin
   end
 
+  def tester_trainer; end
```
Comment (carolyncole) [inline app/models/user.rb:108]: Was this an intended method?

**Review by carolyncole — COMMENTED** [review]
Looks good!  Only one question (see comments)

File: app/models/user.rb (hunk 1 above)
Comment (leefaisonr) [reply app/models/user.rb]: Thanks @carolyncole for the suggested change

**Review by carolyncole — APPROVED** [review]
Thanks!

**Pull request merged**